src/Ng/Jobs/Package2CatalogJob.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
 using System.Net.Http;
```

Comment:
Is this `using` necessary? It doesn't look like you introduced any new types in this file